src/GitHub.VisualStudio/Base/TeamExplorerNavigationItemBase.cs

```diff
@@ -28,7 +28,14 @@ public TeamExplorerNavigationItemBase(ISimpleApiClientFactory apiFactory, ITeamE
             IsVisible = false;
             IsEnabled = true;
 
-            OnThemeChanged();
+            try
+            {
+                OnThemeChanged();
+            }
+            catch (ArgumentNullException)
+            {
+                // This throws in the unit test runner.
+            }
```

Comment:
I think I prefer having this try/catch in the body of the method instead of here.